### PR TITLE
Updated 9anime.to domains to  9anime.id

### PR DESCRIPTION
**Pull request by Abhishek890**
9anime.to domain was blocked for many devices so changed it to 9anime.id

### CODE REVIEW

**Comment by Secozzi** [comment]
.id is just an alternative domain to .to, but not an replacement. You can find all the different domains on .me

**Review by jmir1 — CHANGES_REQUESTED** [review]
You should add the .id domain in the domain preference, not replace the .to domain.
See https://9anime.me for a list of official domains.

**Comment by jmir1** [comment]
> .id is just an alternative domain to .to, but not an replacement. You can find all the different domains on .me

Lmao you were faster 🤣

**Comment by Abhishek890** [comment]
> .id is just an alternative domain to .to, but not an replacement. You can find all the different domains on .me

.to domain is blocked for many isp

**Comment by jmir1** [comment]
We know...
Did you read my review comment?

**Comment by Abhishek890** [comment]
the reason i removed .to is coz its blocked for many isp so i only added those which work mostly everywere

**Review by jmir1 — CHANGES_REQUESTED** [review]

File: src/en/nineanime/src/eu/kanade/tachiyomi/animeextension/en/nineanime/NineAnime.kt [hunk 1]
```diff
@@ -289,8 +289,8 @@ class NineAnime : ConfigurableAnimeSource, ParsedAnimeHttpSource() {
         val domainPref = ListPreference(screen.context).apply {
             key = "preferred_domain"
             title = "Preferred domain (requires app restart)"
-            entries = arrayOf("9anime.to", "9anime.gs", "9anime.pl")
-            entryValues = arrayOf("https://9anime.to", "https://9anime.gs", "https://9anime.pl")
+            entries = arrayOf("9anime.id", "9anime.gs", "9anime.pl", "9anime.id")
```
Comment (jmir1) [inline src/en/nineanime/src/eu/kanade/tachiyomi/animeextension/en/nineanime/NineAnime.kt:292]:
```suggestion
            entries = arrayOf("9anime.to", "9anime.gs", "9anime.pl", "9anime.id")
```
Typo maybe

**Review by jmir1 — CHANGES_REQUESTED** [review]

**Review by jmir1 — CHANGES_REQUESTED** [review]
Now just bump the version code in build.gradle

**Comment by Abhishek890** [comment]
i think its done now and sorry i took so much time to understand

**Review by jmir1 — APPROVED** [review]

**Comment by Abhishek890** [comment]
the issue is still not fixed i think we need to remove that.to extension or replace it with another as after updating the extension it still not work for me 

if i am wrong please correct me

WhatsApp Image 2023-02-21 at 2 57 45 AM

**Comment by jmir1** [comment]
Go to the extension's settings and switch the domain

**Pull request merged**